app/controllers/responses_controller.rb

```diff
@@ -7,7 +7,6 @@ class ResponsesController < ApplicationController
 
   def new
     @response = Response.new
-
     @questions = Question.order("RANDOM()") + DemographicQuestion.first(1)
```

Comment:
Do we still want to hard code the first Demographic Question? I know we aren't finished yet with the work we are doing, but currently we are only importing question types which we can handle with the seeds import script.

Reply:
Great spot, but let's leave it for now as we need to make sure the demographic questions have all their options and question types in the seeds file before we can open it up to all questions